datastore/sqlite_datastore.nim

```diff
@@ -33,53 +39,149 @@ type
     containsStmt: ContainsStmt
     deleteStmt: DeleteStmt
     env: SQLite
+    getDataCol: BoundDataCol
     getStmt: GetStmt
     putStmt: PutStmt
     readOnly: bool
 
 const
-  IdType = "TEXT"
-  DataType = "BLOB"
-  TimestampType = "INTEGER"
-
   dbExt* = ".sqlite3"
-  tableTitle* = "Store"
+  tableName* = "Store"
+
+  idColName = "id"
+  dataColName = "data"
+  timestampColName = "timestamp"
```

Comment:
I think that idColName, dataColName and timestampColName should be exported too, otherwise idCol... procedures aren't very useful

Reply:
It's rather ironic that INSIDE the module you use idColName/... to build sql statements, but expect that external users will use fixed literals

Reply:
I introduced constants for the column names because the same strings / string-fragments were appearing on multiple lines in multiples contexts in the same module, especially with the introduction of `idCol`, etc. That was the whole motivation.

Like I said, it *is* better to export the constants because careful users will want to use them, and next time I will remember that up front. I still anticipate that because the schema is small and/or because many external users will experiment/build custom queries with [DB Browser for SQLite](https://sqlitebrowser.org/) or the `sqlite3` command-line (that's what I would do), where there are no Nim constants but only names visible in the schema, those custom queries will end up embedding the column names when they're copy-pasted into the external user's `.nim` module. So I don't see anything ironic here, I think we just have different ways of thinking about it, which combined will hopefully end up with a good result.